Makefile

```diff
@@ -238,8 +242,12 @@ endif
 # OMPVV static library
 ##################################################
 
-OMPVVLIB=-L$(CURDIR)/ompvv -lompvv
-OMPVVLIB_DEP=$(CURDIR)/ompvv/libompvv.a
+OMPVVLIB=
+OMPVVLIB_DEP=
+ifdef LINK_OMPVV_LIB
+	OMPVVLIB=-L$(CURDIR)/ompvv -lompvv
+	OMPVVLIB_DEP=$(CURDIR)/ompvv/libompvv.a
+endif
```

Comment:
try linking only to the test that uses it

Reply:
The library is now only linked to the test that uses it. Changes are all tested and working.